examples/reporting/get_campaign_stats_to_csv.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This examples gets a basic campaign stats report for a given customer ID
+and writes them to a CSV file.
```

Comment:
This comment should fit on a single line, with any additional information going on a new line, for example:

```
"""This example creates a billing setup for a customer.

A billing setup is a link between a payments account and a customer. The new  
...
```

I think the best thing would be to rephrase the top statement so it fits on one line. Something like:

`Shows how to write data from a basic campaign stats report to a CSV file.`

Reply:
Done.